sdcm/utils/adaptive_timeouts/load_info_store.py

```diff
@@ -249,6 +252,60 @@ def get(self, operation: str | None, timeout_occurred: bool | None = None):
         return [hit["_source"] for hit in res["hits"]["hits"]]
 
 
+class ArgusAdaptiveTimeoutStore(AdaptiveTimeoutStore):
+    """
+    Report adaptive timeout results to Argus.
+    """
+
+    def __init__(self):
+        self.test_config = TestConfig()
+        self.cycle_counters = defaultdict(count)
+
+    def send_adaptive_timeout_results_to_argus(self, results: dict):
+        argus_client = self.test_config.argus_client()
+        if not argus_client:
+            LOGGER.warning("Will not submit to argus - no client initialized")
+            return
+
+        class AdaptiveTimeoutResultsTable(GenericResultTable):
+            class Meta:
+                name = f"{results['operation']} - Timeout Statistics"
+                description = "measurement of specific operation timeouts (ex. decommission, adding nodes etc.)"
+                Columns = [
+                    ColumnMetadata(name="duration", unit="HH:MM:SS", type=ResultType.DURATION, higher_is_better=False),
+                    ColumnMetadata(name="timeout", unit="HH:MM:SS", type=ResultType.DURATION),
+                    ColumnMetadata(name="end_time", unit="", type=ResultType.TEXT),
+                ]
+        cycle = next(self.cycle_counters[results['operation']]) + 1
+        table = AdaptiveTimeoutResultsTable()
+        table.add_result(column="duration", row=f"#{cycle}", value=results['duration'], status=Status.UNSET)
+        table.add_result(column="timeout", row=f"#{cycle}", value=results["timeout"], status=Status.ERROR if results.get(
+            "timeout_occurred") else Status.UNSET)
+        table.add_result(column="end_time", row=f"#{cycle}", value=results["end_time"], status=Status.UNSET)
+        logging.debug("Submitting adaptive timeout results to Argus: %s", table.as_dict())
+        submit_results_to_argus(argus_client, table)
```

Comment:
Just mentioning for awareness: I was thinking we should catch here failures and rollback operation's cycle counter, but actually, for comparisons across runs it will be better to just miss cycle (so graph draws more correct lines). Maybe worth to add comment somewhere around cycles counter

Reply:
btw. I created issue around one possible error: https://github.com/scylladb/argus/issues/680

Reply:
Graphs from those might not be really helpful, since nemesis can be shuffled from time to time.

We'll need overview dashboard for those